tests/mqtt-frames-xpdu/test.rules

```diff
@@ -0,0 +1,4 @@
+alert mqtt any any -> any any (msg:"mqtt frame: pdu 1"; frame:pdu; content: "|a2 0b 00 02|"; startswith; bsize:26; sid:1;)
```

Comment:
This should be `bsize: 13` cf Wireshark highlighting `a20b0002000006746f70696358`